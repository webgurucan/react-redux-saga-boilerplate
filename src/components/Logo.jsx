import React from 'react';
import styled from 'styled-components';

import { ReactComponent as Icon } from 'assets/media/brand/icon.svg';

export const Logo = styled.div`
  align-items: flex-start;
  display: inline-flex;
  font-size: 0;

  svg {
    height: auto;
    max-height: 100%;
    width: 4.8rem;
  }
`;

export default () => (
  <Logo>
    <Icon />
  </Logo>
);
