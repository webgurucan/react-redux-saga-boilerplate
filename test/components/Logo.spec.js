import React from 'react';
import expect from 'expect';
import { mount } from 'enzyme';
import Logo from 'components/Logo';

describe('Logo', () => {
  const wrapper = mount(<Logo />);

  it('should be a Component', () => {
    expect(wrapper.instance()).toBeA(React.Component);
  });

  it('should render properly', () => {
    expect(wrapper.find('.isvg').length).toBe(1);
  });
});
