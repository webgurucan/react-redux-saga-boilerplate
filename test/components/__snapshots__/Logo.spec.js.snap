exports[`Logo should render properly 1`] = `"<div class=\"app__logo\"><span class=\"isvg failed\"></span></div>"`;
